docs/static/breaking-changes.asciidoc

```diff
@@ -3,54 +3,86 @@
 
-**Application Settings:** Introduced a new way to configure application settings for Logstash through a settings.yml file. This file
+[float]
+=== Changes in Logstash Core
+
+* **Application Settings:** Introduced a new way to configure application settings for Logstash through a settings.yml file. This file
 is typically located in `LS_HOME/config`, or `/etc/logstash` when installed via packages. Logstash will not be able
 to start without this file, so please make sure to pass in `--path.settings` if you are starting Logstash manually
 after installing it via a package (RPM, DEB).
 
-**Release Packages:** When Logstash is installed via DEB, RPM packages, it uses `/usr/share/logstash` and `/var/lib/logstash` to install binaries and config files
+* **Release Packages:** When Logstash is installed via DEB, RPM packages, it uses `/usr/share/logstash` and `/var/lib/logstash` to install binaries and config files
 respectively. Previously it used to install in `/opt` directory. This change was done to make the user experience
 consistent with other Elastic products. Full directory layout is described in <<dir-layout>>.
 
-**Command Line Interface:** Most of the long form <<command-line-flags,options>> have been renamed
+* **Default Logging Level:** Changed the default log severity level to INFO instead of WARN to match Elasticsearch. Existing logs
+(in core and plugins) were too noisy at INFO level, so we had to audit log messages and switch some of them to DEBUG
+level.
+
+* **Command Line Interface:** Most of the long form <<command-line-flags,options>> have been renamed
 to adhere to the yml dot notation to be used in the settings file. Short form options have not been changed.
 
-**Plugin Manager Renamed:** `bin/plugin` has been renamed to `bin/logstash-plugin`. This change was to mainly prevent `PATH` being polluted when
+* **Plugin Manager Renamed:** `bin/plugin` has been renamed to `bin/logstash-plugin`. This change was to mainly prevent `PATH` being polluted when
 other components of the Elastic stack are installed on the same instance. Also, this provides a foundation
 for future change which will allow Elastic Stack packs to be installed via this script.
 
-**Kafka Input/Output Configuration Changes:** This release added support for the new 0.9 consumer/producer API which supports security features introduced by Kafka.
+[float]
+=== Breaking Changes in Plugins
+
+* **Elasticsearch Output Index Template:** The index template for 5.0 has been changed to reflect {ref}breaking_50_mapping_changes.html[Elasticsearch's mapping changes]. Most
+importantly, the subfield for string multi-fields has changed from `.raw` to `.keyword` to match Elasticsearch's default
+behavior. The impact of this change to various user groups is detailed below:
+
+* New Logstash 5.0 and Elasticsearch 5.0 users - subfields use `.keyword` from the outset. In Kibana, you can use
```

Comment:
This should be a sub-list of the previous item (right now, it's at the same level).